File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/Machine.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Contains information about a machine.
 * @param id ID of the machine.
 * @param name Name of the machine.
 * @param notes Notes about the machine
 */
data class Machine (

        @get:NotNull 
        @JsonProperty("id") val id: Long,

        @JsonProperty("name") val name: String? = null,

        @JsonProperty("notes") val notes: String? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/Machine.scala

package org.openapitools.client.model


case class Machine (
    /* ID of the machine. */
    _id: Long,
    /* Name of the machine. */
    _name: Option[String],
    /* Notes about the machine */
    _notes: Option[String]
)
object Machine {
    def toStringBody(var_id: Object, var_name: Object, var_notes: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name,"notes":$var_notes
        | }
        """.stripMargin
}
